src/Cli/dotnet/commands/dotnet-workload/install/NetSdkMsiInstallerClient.cs

```diff
@@ -182,6 +185,8 @@ public void GarbageCollect(Func<string, IWorkloadResolver> getResolverForWorkloa
             }
         }
 
+        public string GetFailingWorkloadFromTest() => null;
```

Comment:
Is this only intended for testing? If so, consider making it internal and add the InternalsVisibleToAttribute if it's not already set on the project to which this file belongs

Reply:
Yes, only for testing. Note that this whole class is internal, so this being marked 'internal' instead of 'public' doesn't actually affect anything, and since it's implementing the (also internal) IInstaller interface, it has to be public.